tests/bwc/test_rolling_upgrade.py

```diff
@@ -13,7 +13,7 @@
     UpgradePath('4.4.x', '4.5.x'),
     UpgradePath('4.5.x', '4.6.x'),
     UpgradePath('4.6.x', '4.7.x'),
-    UpgradePath('4.7.x', 'latest-nightly'),
+    UpgradePath('4.7.x', '4.8.x'),
```

Comment:
Shouldn't this keep including latest-nightly? A rolling upgrade from 4 to 5 should be supported if there are no 3.x tables in the cluster

Reply:
ah right